README.md

```diff
@@ -3,18 +3,87 @@
 <table>
   <tr>
     <th>MacOSX (Clang 12.0)</th>
-    <th>Linux</th>
+    <th>Linux (Cuda 11)</th>
+    <th>Linux (gcc 9.3)</th>
   </tr>
   <tr>
     <td><img src="https://img.shields.io/badge/build-failing-red" alt="build-failing"></td>
     <td><img src="https://img.shields.io/badge/build-passing-green" alt="build-failing"></td>
+    <td><img src="https://img.shields.io/badge/build-passing-green" alt="build-failing"></td>
   </tr>
 </table>
 
-1. [Linear Regression](./libtorch-examples/linear-regression)
+## Instructions
+
+### Build and Compile
+
+1. `git clone https://github.com/soumik12345/libtorch-examples --recursive`
 
-2. [Logistic Regression](./libtorch-examples/logistic-regression)
+2. `cmake -DINSTALL_TORCH=true -DCUDA=true -DINSTALL_PYTHON_DEPENDENCIES=true -DDOWNLOAD_MNIST=true ../`
 
-3. [Multi-Layered Perceptron](./libtorch-examples/multi-layered-perceptron)
+    - `-DINSTALL_TORCH` -> Automatically download and install libtorch during build.
+    - `-DCUDA` -> Download GPU Version of LibTorch.
+    - `-DINSTALL_PYTHON_DEPENDENCIES` -> Install Python Dependencies during build.
+    - `-DDOWNLOAD_MNIST` -> Download the MNIST Dataset during build.
+    
+3. `make`
 
-4. [Convnet Classifier](./libtorch-examples/convnet)
+### Run Executables
+
+<table>
+    <tr>
+        <th>Implementation</th>
+        <th>Build Status</th>
+        <th>Run Instructions</th>
+        <th>MNIST</th>
+        <th>CIFAR10</th>
+    </tr>
+    <tr>
+        <td><a href="./libtorch-examples/linear-regression">Linear Regression</a></td>
+        <td><img src="https://img.shields.io/badge/build-passing-green" alt="build-failing"></td>
```

Comment:
Alt text doesn't match. Also use github actions for build shields. I find hard coding shields in bad taste.